node/node.go

```diff
@@ -1079,6 +1080,10 @@ func (n *Node) CreateTransaction(
 		return nil, nil, errors.New("could not fetch signature bytes")
 	}
 
+	if len(value) > n.maxTransactionValueLength {
+		return nil, nil, errors.New("value is too large")
```

Comment:
Why it is not used the error defined in node/errors.go ?

Reply:
defined a new one